tests/components/device_tracker/test_ddwrt.py

```diff
@@ -0,0 +1,250 @@
+"""The tests for the DD-WRT device tracker platform."""
+import os
+import unittest
+from unittest import mock
+import logging
+import requests
+import requests_mock
+
+from homeassistant import config
+from homeassistant.bootstrap import setup_component
+from homeassistant.components import device_tracker
+from homeassistant.const import (CONF_PLATFORM,
+                                 CONF_HOST,
+                                 CONF_PASSWORD,
+                                 CONF_USERNAME)
```

Comment:
Probably personal preference, but most of HASS uses this format, which is a bit more compact (the setup dict is fine to have multi-line, but `import` should probably follow this)

```python
from homeassistant.const import (
    CONF_PLATFORM, CONF_HOST, CONF_PASSWORD, CONF_USERNAME)
```